docs/source/gbd2019_models/causes/maternal_hemorrhage_incidence/index.rst

```diff
@@ -109,220 +111,99 @@ on the ages and sexes to which the cause applies.
 Vivarium Modeling Strategy
 --------------------------
 
-.. todo::
-
-  Add an overview of the Vivarium modeling section.
+Similar to the :ref:`maternal disorders cause model <2019_cause_maternal_disorders>`, we will convert the maternal hemorrhage incidence rate as estimated by GBD in terms of an annual rate among women of reproductive age to events *per birth* (including stillbirths) for use in our :ref:`simulation of IV iron <2019_concept_model_vivarium_iv_iron>`. Births among women of reproductive age in our simulation will be informed by the the :ref:`pregnancy model document <other_models_pregnancy>`.
 
 Scope
 +++++
 
-.. todo::
+This cause model document was developed for the :ref:`IV iron simulation <2019_concept_model_vivarium_iv_iron>`. It is intended to be a DALY-free cause model in order to model in impact on postpartum hemoglobin levels. This cause model will also be affected by maternal anemia level during pregnancy.
 
-  Describe which aspects of the disease this cause model is designed to
-  simulate, and which aspects it is **not** designed to simulate.
+We will not model maternal hemorrhage incidence as a dynamic transition model, but rather as a discrete event that occurs at birth. The probability of maternal hemorrhage incidence will be informed by a ratio per birth derived from GBD data.
 
 Assumptions and Limitations
 +++++++++++++++++++++++++++
 
-.. todo::
-
-  Describe the clinical and mathematical assumptions made for this cause model,
-  and the limitations these assumptions impose on the applicability of the
-  model.
+- We will assume that all causes of maternal hemorrhage occur at the time of birth.
+- Our model of maternal hemorrhage incidence will be measured as cases that do not die due to maternal hemorrhage.
```

Comment:
wouldn't it also count as an incident cases if it *did* lead to death?

Reply:
Yes, that's why in the measure of incidence for this cause model it's measured as (incidence - mortality). However, it is possible that someone had a case of maternal hemorrhage and then died of another maternal disorder and therefore would not experience the disability of living with the hemoglobin deficit due to hemorrhage because they died of something else.